### PR TITLE
:lipstick:(lld): skip drawer and update copy

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - Delete key flow

### 📝 Description

Update delete key flow on LS 
Update copy

https://github.com/user-attachments/assets/0eba5e6a-d1bf-4848-9915-230937add5e1

### ❓ Context

- **JIRA or GitHub link**: [LIVE-14195](https://ledgerhq.atlassian.net/browse/LIVE-14195)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14195]: https://ledgerhq.atlassian.net/browse/LIVE-14195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/static/i18n/en/app.json [hunk 1]
```diff
@@ -6527,10 +6527,10 @@
         }
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-desktop/static/i18n/en/app.json:6527]: you can remove 
```
"options": {
        "deleteBackup": {
          "title": "Delete your backup",
          "description": "All synced accounts will be removed. This won’t affect your local accounts."
        }
      }
```

If I'm not wrong :)

**Review by ypolishchuk-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/static/i18n/en/app.json (hunk 1 above)
Comment (LucasWerey) [reply apps/ledger-live-desktop/static/i18n/en/app.json:6527]: Oh yes missed that

Comment (LucasWerey) [reply apps/ledger-live-desktop/static/i18n/en/app.json]: ✅

File: apps/ledger-live-desktop/static/i18n/en/app.json [hunk 2]
```diff
@@ -6461,12 +6461,12 @@
         "description": "Synchronize your accounts with another Ledger Live app for a seamless experience."
       },
       "backup": {
-        "label": "Manage Key",
-        "description": "Secure or delete your encryption key via another Ledger."
+        "label": "Delete sync",
+        "description": "Your synched crypto accounts across different Ledger Live apps will stop synching."
```
Comment (themooneer) [inline apps/ledger-live-desktop/static/i18n/en/app.json:6465]: **synced** instead of synched

**Pull request merged**